src/baseline_bots/utils.py

```diff
@@ -453,3 +453,18 @@ def smart_select_support_proposals(
             if len(order_list) > 1:
                 optimal_ordering_units.add(ordering_unit)
     return optimal_possible_support_proposals
+
+
+# return a list of powers that are neighbors of power_name
+def neighboring_opps(game, power_name, opponents) -> List[str]:
```

Comment:
This seems like it should be a docstring:

```suggestion
def neighboring_opps(game, power_name, opponents) -> List[str]:
    """Return a list of powers that are neighbors of power_name"""
```

In addition, please add type annotations for the parameters.

Reply:
done